src/sailfishapp_priv.cpp

```diff
@@ -95,9 +95,24 @@ configureApp(QGuiApplication *app)
     //  - Qt Quick Local Storage (QQmlEngine::offlineStoragePath())
 
     QString name = appName();
-    app->setOrganizationName(name);
-    app->setOrganizationDomain(name);
-    app->setApplicationName(name);
+    QString applicationName;
+    QString organizationName;
+    QString organizationDomain;
+
+    MDesktopEntry entry("/usr/share/applications/" + name + ".desktop");
```

Comment:
This would be unfortunate. For now this should commonly work, but going forward we should be getting rid of the requirements that apps go to the specific directories used now. E.g. would be nice if it was possible to have user-specific app installation (without rpm) that would have the .desktop file under user directory.

Reply:
Is using QStandardPaths::ApplicationsPath enough? Although I think it may now use a file in home directory if it happens to exist. It's not necessarily a big issue but I wonder if that could be avoided.

Reply:
Looks a lot nicer already! Think it should now cover the places where it's commony supported to have .desktop files.